General/general_ontology.ttl

```diff
@@ -0,0 +1,22 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
```

Comment:
Can you please do a rebase or merge from main?
This is the reason why the VARIABLES.md file is missing since the code was branched off before we had this feature.

Reply:
merge main into branch - done